projects/1.20/assets/better-archeology/betterarcheology/lang/zh_cn.json

```
@@ -0,0 +1,94 @@
+{
+  "itemGroup.betterarcheology": "更好的考古学",
+
+  "block.betterarcheology.rotten_log": "腐朽的原木",
+  "block.betterarcheology.rotten_planks": "腐朽的木板",
+  "block.betterarcheology.rotten_slab": "腐朽的台阶",
+  "block.betterarcheology.rotten_stairs": "腐朽的楼梯",
+  "block.betterarcheology.rotten_fence": "腐朽的围栏",
+  "block.betterarcheology.rotten_fence_gate": "腐朽的围栏门",
+  "block.betterarcheology.rotten_trapdoor": "腐朽的活板门",
+  "block.betterarcheology.rotten_door": "腐朽的门",
+  "block.betterarcheology.infested_mud_bricks": "感染的泥砖",
+  "block.betterarcheology.cracked_mud_bricks": "破裂的泥砖",
+  "block.betterarcheology.cracked_mud_brick_slab": "破裂的泥砖台阶",
+  "block.betterarcheology.cracked_mud_brick_stairs": "破裂的泥砖楼梯",
+  "block.betterarcheology.suspicious_red_sand": "可疑的红沙",
+  "block.betterarcheology.suspicious_dirt": "可疑的泥土",
+  "block.betterarcheology.fossiliferous_dirt": "化石泥土",
+  "block.betterarcheology.fossiliferous_dirt_tooltip": "也许能够挖出些什么？",
+  "block.betterarcheology.villager_fossil": "村民化石",
+  "block.betterarcheology.villager_fossil_tooltip": "它的化石手臂可以用作一种展示",
+  "block.betterarcheology.villager_fossil_head": "村民化石头颅",
+  "block.betterarcheology.villager_fossil_head_tooltip": "它们似乎没有演化很多...",

```

```suggestion
  "block.betterarcheology.villager_fossil_head_tooltip": "它们似乎没有演化很多……",
```
请注意使用中文省略号，一般在输入法中按下`Shift`+`6`即可